de.json

```diff
@@ -27,7 +27,7 @@
   "board.info.have": "Du hast",
   "board.info.want": "Du möchtest",
   "board.tier.selected": "Level ausgewählt",
-  "board.label.main": "Level zu Level austauschg",
+  "board.label.main": "Tausch mit gleichem Level",
```

Comment:
I have some suggestions here. It is possible to exchange maps without the tier limitations by activating the slider. It would be great to use here something like "map exchange" because tier to tier is not actual anymore.
Also, it would be great to append your name (github or game account or any you would like) to the "contributors.names"